platform/linux/src/headless_backend_egl.cpp

```diff
@@ -68,7 +68,7 @@ class EGLDisplayConfig {
     EGLConfig config = 0;
 };
 
-class EGLBackendImpl : public HeadlessBackend::Impl {
+class EGLBackendImpl final : public HeadlessBackend::Impl {
```

Comment:
I am surprised that you needed to touch a file in the platform/linux folder since this change is mostly about the android test app. Can you comment?